ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor/Event.hs

```diff
@@ -116,6 +116,8 @@ data Event (muxMode :: MuxMode) initiatorCtx peerAddr versionData m a b
     | MaturedDuplexPeers   !(Map peerAddr versionData)         -- ^ newly matured duplex peers
                            !(OrdPSQ peerAddr Time versionData) -- ^ queue of fresh duplex peers
 
+    | InactivityTimeout
```

Comment:
I think that this name is rather confusing.
The governor is just waiting for time to pass. No connections is inactive (if they where they would be reset).
 Can you pick a better name?